nativeauthenticator/handlers.py

```diff
@@ -263,10 +311,13 @@ def validate_slug(slug, key):
 
 
 class ChangePasswordHandler(LocalBase):
-    """Render the reset password page."""
+    """Render the /hub/change-password page where
+    users can change their own password."""
```

Comment:
That makes me think there is only a `get()`, but perhaps we can clarify that it has a get + post, and the post will react to the submission of the initially rendered form.

Reply:
I've made the docstring more explicit and mentioned that the handler is responsible for both GET and POST requests.